### ISSUE TITLE
Installed successfully but can't find interception.dll?

### ISSUE THREAD

**Issue by GreenSleeper** [oblitum/Interception#155]
Command line installter installed successfully but when I reboot, it says can't find interception.dll.
I reinstalled it and the error is still there.

**Comment by GreenSleeper** [comment]
> Command line installter installed successfully but when I reboot, it says can't find interception.dll. I reinstalled it and the error is still there.

It's not a issue about intercept, sorry
I put the exe in startup folder, and it can't find intercept.dll

**Comment by Nico-Mayer** [comment]
Hey @GreenSleeper i have the same problem, how did you fix this?

**Comment by Masca1919** [comment]
Same here. I extracted the installation zip to the Desktop, ran the install-interception.exe from admin powershell (also tried cmd), and got a successful install message in both cases.
However, I run into the "can't find interception.dll" when trying to use the API (with Kanata) and the dll is indeed nowhere to be found (except in the "library" folder from the extracted installation directory).

If I understand @GreenSleeper correctly, it may not be the same issue: I did not move the exe from its place in the extracted installation directory, it is inside the "command line installer" folder.

@Nico-Mayer did you advance in any way ?
